ublox-bluetooth-commands-mode.go

```diff
@@ -58,13 +59,27 @@ func (ub *UbloxBluetooth) ResetUblox() error {
 // ResetUblox calls the Serial port's ResetViaDTR and does not return until
 // the ublox module has indicated it is ready
 func (ub *UbloxBluetooth) ResetUbloxSync() error {
+	ub.rebootExpected = true
```

Comment:
Should this be protected by a mutex? As there is a wait in the function we might get unexpected flag states.

Reply:
If this was a hard real time application then yes. I'd rather have the function return the error, than block on a mutex(). Easier to spot a problem